### PR TITLE
feat/neutron-logger

### PR DESCRIPTION
**Pull request by sotnikov-s**
https://p2pvalidator.atlassian.net/browse/NTRN-192

replace the general zap.Logger used everywhere with personal loggers for all logging entities from
- https://github.com/neutron-org/neutron-logger/pull/1 and https://github.com/neutron-org/neutron-logger/pull/2

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -37,6 +37,5 @@ RELAYER_STORAGE_PATH=storage/leveldb
 RELAYER_QUERIES_TASK_QUEUE_CAPACITY=10000
 RELAYER_CHECK_SUBMITTED_TX_STATUS_DELAY=10
 
-#LOGGER_LEVEL=info
-#LOGGER_OUTPUTPATHS=stdout, /tmp/logs
```
Comment (oopcode) [inline .env.example:41]: wait, we can't specify logger outputs now?

Comment (oopcode) [reply .env.example:41]: i'm not sure that i like the idea of forcing the user to use both env vars and config files if they want to set logger output. also, you'll need to mount additional volumes if you want to do that in docker, which is sort of dirty

**Review by oopcode — CHANGES_REQUESTED** [review]
i'm not sure that i like the idea of forcing the user to use both env vars and config files if they want to set logger output. also, you'll need to mount additional volumes if you want to do that in docker, which is sort of dirty

File: README.md [hunk 2]
```diff
@@ -123,8 +123,7 @@ Relayer:
 
 # Logging
 
-We are using [zap.loger](https://github.com/uber-go/zap)
-By default, project spawns classical Production logger. so if there is a need to customize it, consider editing envs (see .env.example for exapmles)
+We are using a little bit modified [zap.loger](https://github.com/uber-go/zap), the modification can be seen at the [neutron-logger repository](https://github.com/neutron-org/neutron-logger). The default version of the logger used in the application is a little bit modified [zap.NewProduction](https://github.com/uber-go/zap/blob/master/logger.go#L94). The logger level can be set via the `LOGGER_LEVEL` env variable. If there is a need for a more significant customisation of the logger behaviour, see the neutron-logger repository readme (for example, if you want to set the logger output path, you have to provide a config file).
```
Comment (foxpy) [inline README.md:126]: Why do we call it `zap.loger`? Isn't it supposed to be `zap.Logger`?

**Review by foxpy — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (sotnikov-s) [reply README.md]: fixed

File: .env.example (hunk 1 above)
Comment (sotnikov-s) [reply .env.example]: we could, but the realisation was so-so. improved it so the logger is configurable via envs again

**Review by foxpy — COMMENTED** [review]
Looks alright for me. Not sure if I have to approve it considering @oopcode's review.

**Review by foxpy — DISMISSED** [review]
Approving this since GitHub UI doesn't allow me to cancel my request for changes.

**Review by foxpy — APPROVED** [review]

**Review by oopcode — APPROVED** [review]

**Pull request merged**